arch/Cargo.toml

```diff
@@ -15,6 +15,7 @@ vm-memory = { version = "0.1.0", features = ["backend-mmap"] }
 
 acpi_tables = { path = "../acpi_tables", optional = true }
 arch_gen = { path = "../arch_gen" }
+vm-allocator = { path = "../vm-allocator" }
```

Comment:
As mentioned in another comment, let's duplicate the simple `pagesize` function in this crate, instead of adding a dependency on `vm-allocator` just for this.
And long term, I think `pagesize()` should be part of `vmm-sys-util`.

Reply:
done.